docs/doc/90-contributing/09-how-to-write-aggregate-functions.md

```diff
@@ -2,135 +2,126 @@
 title: How to Write Aggregate Functions
 ---
 
+An aggregate function, or aggregation function, is a function that combines values from multiple rows to form a single summary value. Common aggregate functions include sum, count, avg, and others.
+
 ## How to write aggregate functions
 
 Databend allows us to write custom aggregate functions through rust code.
+
 It's not an easy way because you need to be a rustacean first. Databend has a plan to support writing UDAFs in other languages(like js, web assembly) in the future.
```

Comment:
I dont understand the first part.....

for the second part: Databend plans to support writing UDAFs in other languages (like js, web assembly) in the future.